InitializationProtocol/ManagementProtocol.adoc

```diff
@@ -0,0 +1,135 @@
+OmniXtend Management Protocol Documentation
+===========================================
+:Author: Nathaniel Graff
+:Email: <nathaniel.graff@sifive.com>
+
+== Introduction
+
+The OmniXtend Management Protocol describes the process by which a cluster of
+OmniXtend Nodes is configured from cold boot by an OmniXtend Coordinator and
+transitioned to the next stage of the boot flow.
+
+=== Glossary
+
+OmniXtend Cluster::
+	A collection of machines, connected by Ethernet, which includes both OmniXtend Nodes, an OmniXtend Coordinator, and optionally network switches.
+OmniXtend Node::
+	One of several machines in an OmniXtend Cluster which implements the TileLink over Ethernet protocol (TLoE).
+OmniXtend Coordinator::
+	A single machine in the OmniXtend Cluster which performs the Coordinator role in the OmniXtend Management Protocol. The Coordinator may be any machine connected to the same Ethernet network capable of sending and receiving Ethernet packets so that the Coordinator role may be performed by any of a separate physical server, a programmable network switch, or an OmniXtend Node.
+TileLink over Ethernet::
+	The hardware implementation and Ethernet protocol by which Nodes in an OmniXtend cluster share a coherent TileLink Bus. The task of the OmniXtend Management Protocol is to bring up the TileLink over Ethernet hardware configuration of all OmniXtend nodes such that the TileLink over Ethernet Protocol allows the Cluster to execute the next stage of the boot flow without deadlocking.
+TLoE::
+	An Acronym for TileLink over Ethernet
+TileLink::
+	A coherent chip-scale interconnect designed for use in a System-on-Chip to connect general-purpose multiprocessors, co-processors, accelerators, DMA engines, and other devices.
+Devicetree::
+	A markup specification for describing the hardware resources of a computer.
+Devictree Binary::
+	A particular encoding of Devicetree markup designed for machine consumption.
+Single-Node DTB::
+	A Devicetree in Devicetree Binary format which describes the hardware resources of an individual OmniXtend Node
+Cluster DTB::
+	A Devicetree in Devicetree Binary format which describes the hardware resources of the OmniXtend Cluster and contains all of the information necessary for all OmniXtend Nodes to configure and bring-up their TLoE hardware interfaces.
+
+== Management Protocol
+
+This section describes the protocol by which OmniXtend Nodes communicate over Ethernet with an OmniXtend Coordinator to:
```

Comment:
The flow is different for booting node & non-booting node depending on whether NUMA system or distributed system is used. We need to clarify those upfront.We also need to add another diagram for booting node.

Reply:
Once the TLoE links are brought up in the correct order, the system is free to boot however it wants. I don't see how this protocol needs to concern itself with "booting" versus "non-booting nodes.

Reply:
It depends on the scope of this protocol. I thought this protocol would specify the configuration of each node and complete booting sequence. We do need to specify all the possible states that nodes can be configured if we are talking about configuration.

If the scope only involves initialization, we probably don't need it.